src/main/java/com/plivo/api/models/base/Deleter.java

```diff
@@ -40,6 +42,10 @@ public void delete() throws IOException, PlivoRestException {
     return this;
   }
 
+  protected Map<String, Object> toMap() {
+    client();
```

Comment:
Why client() in toMap utility method required ?

Reply:
line not required, reverting it